cypress/utils/utils.ts

```diff
@@ -258,7 +258,7 @@ export function resetURL(): void {
 
 export function selectItemsPerPage(items: number): void {
     cy.log(`Select ${items} per page`);
-    cy.get(commonView.itemsPerPageToggleButton, { timeout: 60 * SEC, log: false }).then(
+    cy.get(commonView.itemsPerPageToggleButton, { timeout: 120 * SEC, log: false }).then(
```

Comment:
Why the 2 mins timeout?

If an operation like clicking that button takes 2 min that's probably due to a bug and we'd like it to fail

Reply:
I added more timeout because the page takes some time to load the elements, is that a bug ?

Reply:
I think so, yes, the page can take some time to load the elementes, but it can't take 2 mins.

Reply:
Thanks @midays @abrugaro. I guess first thing to cross-check is environment(cluster) issue and second if it takes more than 60 secs to load we can log it as a bug.

Reply:
Thanks for the feedback, I've triggered a [jenkins test run](https://main-jenkins-csb-migrationqe.apps.ocp-c1.prod.psi.redhat.com/job/mta/job/mta-ui-tests-runner/3319/) with the timeout reduced back to 60 seconds, and it passed, so I guess we're good
@abrugaro @kpunwatk